demo/webpack.config.dev.js

```diff
@@ -33,13 +33,13 @@ module.exports = {
       {
         test: /\.jsx?$/,
         exclude: [/node_modules/],
-        loaders: ["babel-loader?stage=0"]
+        loader: "babel"
```

Comment:
I switched all these loaders to use the shorthand form. They are equivalent, but the shorthand form seems to be more common  in docs (checked `webpack` itself, `babel-loader` and `css-loader` GitHub pages).

(The fact that you can specify both has always driven me nuts. I always thought just having "babel" here meant it was somehow using the "babel" npm package instead of "babel-loader". Of course that's not actually the case but it's confusing to Webpack beginners.)